### PR TITLE
feat: create wallet flow

### PR DESCRIPTION
**Pull request by magiziz**
# Description

Please include a brief summary of the change.

## Type of change

- [x] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Associated Issues

For Linear issues: Closes APKT-831

# Showcase (Optional)

&nbsp;

&nbsp;

# Checklist

- [x] Code in this PR is covered by automated tests (Unit tests, E2E tests)
- [x] My changes generate no new warnings
- [x] I have reviewed my own code
- [x] I have filled out all required sections
- [x] I have tested my changes on the preview link
- [ ] Approver of this PR confirms that the changes are tested on the preview link

### CODE REVIEW

File: packages/scaffold-ui/src/partials/w3m-email-login-widget/index.ts [hunk 1]
```diff
@@ -79,12 +79,7 @@ export class W3mEmailLoginWidget extends LitElement {
         ${this.submitButtonTemplate()}${this.loadingTemplate()}
         <input type="submit" hidden />
       </form>
-
-      ${socials || !multipleConnectors || !enableWallets
-        ? null
-        : html`<wui-flex .padding=${['xxs', '0', '0', '0'] as const}>
-            <wui-separator text="or"></wui-separator>
-          </wui-flex>`}
+      ${this.separatorTemplate()}
```
Comment (magiziz) [inline packages/scaffold-ui/src/partials/w3m-email-login-widget/index.ts:82]: Created another function to separate the logic a bit

File: packages/scaffold-ui/src/views/w3m-connect-view/index.ts [hunk 2]
```diff
@@ -76,6 +100,35 @@ export class W3mConnectView extends LitElement {
...
 
+  private guideTemplate() {
+    const socials = this.features?.socials
+    const enableWallets = OptionsController.state.enableWallets
+
+    const socialsExist = socials && socials.length
+
+    if (!this.authConnector && !socialsExist) {
+      return null
+    }
```
Comment (magiziz) [inline packages/scaffold-ui/src/views/w3m-connect-view/index.ts:111]: We'll not show the wallet guide UI unless user has enabled email or social login

File: packages/ui/src/composites/wui-chip/index.ts [hunk 3]
```diff
@@ -18,6 +18,10 @@ export class WuiChip extends LitElement {
 
   @property() public imageSrc?: string = undefined
 
+  @property() public imageIcon?: IconType = undefined
+
+  @property() public imageIconSize: SizeType = 'md'
```
Comment (magiziz) [inline packages/ui/src/composites/wui-chip/index.ts:23]: I added two properties so i can use icons as well. 

At the moment the `<wui-chip>` component only accepts `imageSrc` property which requires me to use `https://*` only

File: packages/ui/src/assets/svg/walletconnect.ts [hunk 4]
```diff
@@ -11,3 +11,17 @@ export const walletConnectSvg = svg`<svg fill="none" viewBox="0 0 96 67">
     d="M25.67 19.15a32.06 32.06 0 0 1 44.66 0l1.5 1.48c.43.42.43 1.09 0 1.5l-5.15 5.05a.31.31 0 0 1-.44 0l-2.07-2.03a23.21 23.21 0 0 0-32.34 0l-2.22 2.18a.31.31 0 0 1-.44 0l-5.15-5.05a1.05 1.05 0 0 1 0-1.5l1.65-1.63ZM81 29.6l4.6 4.5c.42.41.42 1.09 0 1.5l-20.7 20.26c-.43.43-1.14.43-1.57 0L48.63 41.47a.9.9 0 0 0-1.26 0L32.68 55.85c-.43.43-1.14.43-1.57 0L10.42 35.6a1.05 1.05 0 0 1 0-1.5l4.59-4.5a1.13 1.13 0 0 1 1.57 0l14.68 14.38a.9.9 0 0 0 1.27 0l-.35-.35.35.35L47.22 29.6a1.13 1.13 0 0 1 1.56 0l14.7 14.38a.9.9 0 0 0 1.26 0L79.42 29.6a1.13 1.13 0 0 1 1.57 0Z"
   />
 </svg>`
+
+export const walletConnectLightBrownSvg = svg`
```
Comment (svenvoskamp) [inline packages/ui/src/assets/svg/walletconnect.ts:15]: Can't we reuse the other svg in this file? It has `currentColor` as fill

**Review by svenvoskamp — APPROVED** [review]

**Pull request merged**